cadc-vos/src/main/java/ca/nrc/cadc/vos/VOS.java

```diff
@@ -235,6 +235,9 @@ else if (value.equals(properties.getValue()))
      * Standard Node Properties defined by the CADC
      */
 
+    // Pass information to the service, e.g. runid = "TEST" means the service call is a test
```

Comment:
Could you please change the comment to:
"Property used for identifying a transaction"